app-emulation/cloud-init/cloud-init-9999.ebuild

```diff
@@ -33,15 +33,15 @@ CDEPEND="
 	dev-python/requests[${PYTHON_USEDEP}]
 	dev-python/jsonpatch[${PYTHON_USEDEP}]
 	dev-python/jsonschema[${PYTHON_USEDEP}]
-	dev-python/six[${PYTHON_USEDEP}]
 "
 DEPEND="
 		${CDEPEND}
 	test? (
 		>=dev-python/httpretty-0.7.1[${PYTHON_USEDEP}]
 		dev-python/mock[${PYTHON_USEDEP}]
-		dev-python/nose[${PYTHON_USEDEP}]
-		dev-python/coverage[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-cov[${PYTHON_USEDEP}]
+		dev-python/setuptools[${PYTHON_USEDEP}]
 	)
 "
```

Comment:
I can't comment on the right line but I'm guessing the `distutils_enable_tests nose` line needs to become `distutils_enable_tests pytest`?

(Also, if so, we can drop pytest from the test dependencies, as it's implied by `d_e_t`.)

Reply:
I missed that, I suspect you're right:
https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html

I'll try your suggestion.

Reply:
https://projects.gentoo.org/python/guide/distutils.html#enabling-tests is a good resource too